Information/CenterOfMass.FCMacro

```diff
@@ -250,11 +270,11 @@ class CenterofmassWidget(QtGui.QWidget):
         label_comboUnit.setAlignment(QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter)
 
         comboUnitDensity = QtGui.QComboBox(toolTip='Unit of density')
-        comboUnitDensity.addItems(['kg/m³',
-                                   'g/dm³', 'g/cm³',
-                                   'mg/mm³',
-                                   'oz/in³',
-                                   'lb/in³', 'lb/ft³', 'lb/yd³'])
+        comboUnitDensity.addItems(['kg/m^3',
```

Comment:
The original was nicer. Why this change?

Reply:
I have a strange (Â) that is appearing sometimes. The ^3 seems more "stable"